tmk_core/protocol/vusb/vusb.c

```diff
@@ -71,6 +79,52 @@ void vusb_transfer_keyboard(void) {
     }
 }
 
+/*------------------------------------------------------------------*
+ * RAW HID
+ *------------------------------------------------------------------*/
+#ifdef RAW_ENABLE
+#    define RAW_INPUT_SIZE (32)
+#    define RAW_OUTPUT_SIZE (32)
```

Comment:
```suggestion
#    define RAW_EPSIZE 32
```
We really only need the one here.

Reply:
EPSIZE is limited to "8" by VUSB.   
If we need simpler, what do you think of this?

```suggestion
#    define RAW_BUFFER_SIZE 32
#    define RAW_EPSIZE 8
```

Reply:
Ah, this seems to be a limitation of USB 1.1 itself, not V-USB:
https://www.beyondlogic.org/usbnutshell/usb4.shtml#Interrupt
> The maximum data payload size for low-speed devices is 8 bytes.

So yes, `RAW_EPSIZE` should be 8. Not sure what the buffer size would be used for?

Reply:
> Ah, this seems to be a limitation of USB 1.1 itself, not V-USB:

But most hosts will be able to accept it.

`RAW_BUFFER_SIZE` is used to check the report length for compatibility with other protocols, as shown below.
```C
void raw_hid_send(uint8_t *data, uint8_t length) {
    if (length != RAW_BUFFER_SIZE) {
...
raw_hid_receive(raw_output_buffer, RAW_BUFFER_SIZE);
```

Reply:
Right, so we would set the endpoint size to 8 and the report size to 32? That sounds fair enough.

Reply:
> we would set the endpoint size to 8 and the report size to 32?
Yes, I'll work on this tonight.

Reply:
I finished! Does this look good?

Reply:
Yup!